lib/literature/views/error_view.ex

```diff
@@ -13,4 +14,68 @@ defmodule Literature.ErrorView do
   def template_not_found(template, _assigns) do
     Phoenix.Controller.status_message_from_template(template)
   end
+
+  def render("404.html", assigns) do
+    ~H"""
+    <.error_page>
+      <:error_code>404</:error_code>
+      <:title>Page not found.</:title>
+      <:message>Sorry, we couldn't find the page you are looking for.</:message>
+    </.error_page>
+    """
+  end
+
+  def error_page(assigns) do
+    ~H"""
+    <div
+      class="error-content py-12 flex flex-col bg-white"
+      id="error-page-component"
+    >
+      <main class="flex-grow flex flex-col justify-center max-w-7xl w-full mx-auto px-4 sm:px-6 lg:px-8">
```

Comment:
@technicalcapt this should not be martide specific so can we please remove and just use the phoenix default?

Reply:
@samhamilton Do you mean to remove the martide icon below in the error content?

Reply:
https://github.com/martide/literature/pull/128/files#diff-3b814f6c184626a7bc0753d5cecf97be7cb5ad815c4d10da7fa05d0296b29eccR35-R40

Reply:
it should just be using the built in page and not report an error to sentry

Reply:
Yes, when we're adding this error, it should show the 404 page and not report to Sentry.